spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -0,0 +1,490 @@
+#!/usr/bin/env python
+#########################################################################################
+#
+# Compute maximum spinal cord compression using AP diameter or other morphometrics.
+#
+# ---------------------------------------------------------------------------------------
+# Copyright (c) 2015 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Author: Sandrine Bédard, Jan Valosek, Julien Cohen-Adad
+#
+# About the license: see the file LICENSE.TXT
+#########################################################################################
+# TODO: add option to normalize or not
+# TODO: maybe create an API or move some functions
+import sys
+import os
+import numpy as np
+import csv
+import logging
+from typing import Sequence
+import pandas as pd
+from spinalcordtoolbox.utils import SCTArgumentParser, Metavar, init_sct, set_loglevel
+from spinalcordtoolbox.utils.fs import get_absolute_path, check_file_exist, extract_fname, printv
+from spinalcordtoolbox.image import Image
+from spinalcordtoolbox import __data_dir__
+
+
+logger = logging.getLogger(__name__)
+
+
+# PARSER
+# ==========================================================================================
+def get_parser():
+    parser = SCTArgumentParser(
+        description='Compute spinal cord compression morpometrics such as Maximum Spinal Cord Compression as in'
+                    'Miyanji F, Furlan JC, Aarabi B, Arnold PM, Fehlings MG. Acute cervical traumatic spinal cord injury:'
+                    ' MR imaging findings correlated with neurologic outcome--prospective study with 100 consecutive patients.'
+                    ' Radiology 2007;243(3):820-827.'
+                    'Can be computed using AP diameter or other morphometrics (CSA, RL diameter, eccentricity and solidity).'
+                    'Morphometrics are normalized with a reference database where morphometrics from sct_process_segmentation '
+                    'are brought to PAM50 anatomical space using -normalize-PAM50.'
```

Comment:
```suggestion
        description='Compute spinal cord compression morphometrics such as the Maximum Spinal Cord Compression [1], '
                    'the antero-posterior (AP) diameter, or other relevant spinal cord morphometrics that are output'
                    ' by the function sct_process_segmentation (CSA, RL diameter, eccentricity, solidity, etc.).'
                    ' Metrics are normalized using a database of spinal cord morphometrics built from healthy control'
                    ' subjects. This database uses the PAM50 template as an anatomical reference system.'
                    
                    '[1]: Miyanji F, Furlan JC, Aarabi B, Arnold PM, Fehlings MG. Acute cervical traumatic spinal cord injury:'
                    ' MR imaging findings correlated with neurologic outcome--prospective study with 100 consecutive' 
                    ' patients. Radiology 2007;243(3):820-827.'
```

Reply:
I also suggest you add documentation about how one can interpret a result like this:

|   Subject             |   Compression Level  |   MEAN(diameter_AP) ratio  |   Normalized MEAN(diameter_AP) ratio  |
|-----------------------|----------------------|----------------------------|---------------------------------------|
|   sub-twh018_T2w.csv  |   4                  |   2.4655002897446400       |   10.375583376646100                  |
|   sub-twh018_T2w.csv  |   5                  |   7.177562376846180        |   11.525718035912400                  |
|   sub-twh018_T2w.csv  |   7                  |   4.854757889876880        |   1.8346037155670000                  |

Ie: explain what "MEAN(diameter_AP) ratio" is (eg: 1 means no compression would, 2 means that ...). 

Also, it would be nice to include the equation of the ratio, and the normalization in the usage.